### PR TITLE
Fix directory structure

### PR DESCRIPTION
**Pull request by t-asaka**
- Abolished version directory
- Moved awsiotprotocol, pubqueue, subqueue packages to internal directory
- Go support version 1.11 or later

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,24 +1,20 @@
...
-  - "1.10"
   - "1.11"
+  - "1.12"
   - master
 
 env:
   global:
     - REVIEW_DOG_ON=1.11
     - REVIEW_DOG_VERSION=0.9.11
-    - VERSION_DIR=v2
```
Comment (at-wat) [inline .travis.yml]: `- GO111MODULE=on` is needed for 1.11 and 1.12.
`on` will be default on 1.13 (master).

**Comment by t-asaka** [comment]
@at-wat please take a look

File: .travis.yml [hunk 2]
```diff
@@ -45,7 +37,6 @@ script:
       golint ./... | reviewdog -f=golint -reporter=github-pr-review
```
Comment (at-wat) [inline .travis.yml]: above `cd ${TRAVIS_BUILD_DIR}` can be removed.

File: internal/awsiotprotocol/mqtts_test.go [hunk 3]
```diff
@@ -30,9 +30,9 @@ func TestMqttsNewClientOptions(t *testing.T) {
 	opt := &Config{
 		URL:      "mqtts://example.com:8882",
 		ClientID: "mqttsclientid",
-		CaPath:   "../sample/samplecerts/cafile.pem",
-		CertPath: "../sample/samplecerts/client-crt.pem",
-		KeyPath:  "../sample/samplecerts/client-key.pem",
+		CaPath:   "../.../sample/samplecerts/cafile.pem",
```
Comment (at-wat) [inline internal/awsiotprotocol/mqtts_test.go]:
```suggestion
		CaPath:   "../../sample/samplecerts/cafile.pem",
```

File: .travis.yml [hunk 4]
```diff
@@ -45,7 +37,6 @@ script:
...
     fi
 after_success:
-  - cd ${TRAVIS_BUILD_DIR}/${VERSION_DIR}
   - |
```
Comment (at-wat) [inline .travis.yml]: if statement looks no longer needed.
```suggestion
  - bash <(curl -s https://codecov.io/bash)
```
and removing `REPORT_COVERAGE` would be better.

File: .travis.yml [hunk 5]
```diff
@@ -1,54 +1,39 @@
...
   - go build
   - |
-    if [[ ! ${TRAVIS_GO_VERSION} =~ ^1\.(8|9)(\.[0-9]+){0,1}$ ]]
-    then
-      echo "Enable coverage report"
-      REPORT_COVERAGE=1
-      TEST_ARG="-coverprofile=coverage.txt -covermode=atomic -coverpkg=all"
-    fi
+    echo "Enable coverage report"
+    REPORT_COVERAGE=1
```
Comment (at-wat) [inline .travis.yml]:
```suggestion
```

Comment (t-asaka) [reply .travis.yml]: :bow:

**Pull request merged**